source/hwIo/ioThread.py

```diff
@@ -17,6 +17,7 @@
 from inspect import ismethod
 from buildVersion import version_year
 import NVDAState
+from watchdog import getFormattedStacksForAllThreads
```

Comment:
Since it is not used only for watchdog, would it be worth to move `getFormattedStacksForAllThreads` in a log-related file? E.g. `logHandler.py`

With `watchdog.getFormattedStacksForAllThreads` remaining but being deprecated.

Reply:
I think that makes sense, but then we could also extend the stack_info argument to take something to log all stacks. That might be something more suitable for a follow up.